virt/virt-2-6-release-notes.adoc

```diff
@@ -61,6 +61,9 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //CNV-9455: Improvements to KubeVirt and CDI ensure that storage is allocated on the correct node
 
 //CNV-9456: When importing VM disk images, you can now control which multus network is used
+* You can now control which network the Containerized Data Importer (CDI) uses when importing virtual machine disk images by adding one of the following annotations to the `metadata.annotations` field of a `DataVolume` object:
+`k8s.v1.cni.cncf.io/networks: <network_name>`:: If you use this annotation, the transfer pods use both the default network from the cluster and the secondary Multus network.
+`v1.multus-cni.io/default-network: <network_name>`:: If you use this annotation, the transfer pods use the Multus network by default.
```

Comment:
@arnongilboa Can you review this please?  Do we need to mention both here?  I thought only the second one listed above is effective.

Reply:
I would mention only `k8s.v1.cni.cncf.io/networks`, as it's the one which was e2e tested.

Reply:
Maybe it's better to mention both of them, since the two annotations have different usage.

Reply:
@duyanyan But have we really e2e tested the default-network except verifying it is passed to the pods?

Reply:
I just thought we mentioned both in our upstream doc, do we need to keep it consistent?

Reply:
I think its test was not e2e as the other annotation, so we cannot guarantee for it's actual result. However we experimented with it enough to document it upstream. @aglitke ?

Reply:
@duyanyan and @arnongilboa Anything we document downstream is 'supported' by Red Hat (unless it is designated tech/developer preview, which CNV is not). I will remove the `v1.multus-cni.io/default-network` annotation from this doc. Thanks for the info!